cbor/examples/cbor/main/idf_component.yml

```diff
@@ -0,0 +1,3 @@
+## IDF Component Manager Manifest File
+dependencies:
+  espressif/cbor: ">=0.5.4,<1.0.0"
```

Comment:
We should consider adding `override_path` here, please refer to https://github.com/espressif/idf-component-manager/wiki#adding-dependency-on-the-component-for-examples

Reply:
Added `override_path` in idf_component.yml file.